app/src/main/res/values/string-not_translatable.xml

```diff
@@ -10,6 +10,7 @@ work. If not, see <https://creativecommons.org/publicdomain/zero/1.0/>.
 -->
 <resources>
     <string name="app_name" translatable="false">Markor</string>
+    <string name="app_fileprovider" translatable="false">net.gsantner.markor.provider</string>
```

Comment:
this string can be removed (as not needed)


Use this (can be copy-pasted from `markor/ShareUtil.java`) below:
```
    public static final String FILE_PROVIDER_AUTHORITY = BuildConfig.APPLICATION_ID + ".provider";
```